app/main.py

```
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) == 3 and parts[0] == "cp" and parts[1] != parts[2]:
+        cp, old_file, new_file = parts
+        try:
+            with (open(old_file, "r") as file_in,
+                  open(new_file, "w") as file_out):
+                file_out.write(file_in.read())
+        except Exception:
+            pass
```

Using `except Exception: pass` is generally discouraged because it can hide important errors. For instance, if the source file doesn't exist (`FileNotFoundError`), this code will silently fail, and the user won't know why. It's better to handle specific exceptions or to let the program raise an error if an unexpected issue occurs.